app/http/middleware/CORSMiddleware.py

```diff
@@ -0,0 +1,34 @@
+""" CORS Middleware """
+
+from masonite.request import Request
+
+
+class CORSMiddleware:
+    """CORS Middleware
+    |--------------------------------------------------------------------------
+    | Create Dictionary of CORS headers
+    |--------------------------------------------------------------------------
+    |
+    | Appends CORS headers to HTTP response
+    |
+    | Ex. cors = {"Access-Control-Allow-Origin": "*",
+    |             "Access-Control-Allow-Headers": "Content-Type",
+    |             "Access-Control-Allow-Methods": "DELETE, GET, HEAD, OPTIONS, PATCH, POST, PUT"}
+    |
+    """
+
+    CORS = {}
```

Comment:
if we go on `Django` style, we can configure like this

```python
CORS = {"Access-Control-Allow-Origin": "*"}
```

Personally I don't like it. But if you think it's better do it, i can commit the update. Just tell me :)